### DISCUSSION TITLE
present custom events and present custom notes

### DISCUSSION THREAD

**Discussion by KulKid** [YoshiCrafter29/YoshiCrafterEngine#141] [Ideas]
please add those
so people with no coding skill can get an example of a custom event
or a custom note

**Comment by dimuzicle** [comment]
custom notes are existing
and events are here too

**Reply by FridayNightGachaTrap** [reply]
They meant preset notes & events

**Comment by KulKid** [comment]
i dont know how to make them myself
or use them
there is no youtube tutorials
